src/seedu/addressbook/data/person/Address.java

```diff
@@ -27,6 +38,16 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
         this.value = trimmedAddress;
```

Comment:
Is there still a need to keep the address string, or can you simply reconstruct the address on the fly?